Sources/Client/Client/Client.swift

```diff
@@ -203,6 +203,21 @@ public final class Client {
             webSocket.disconnectInBackground()
         }
     }
+    
+    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases), channelId: String? = nil, callback: @escaping OnEvent) -> Subscription {
```

Comment:
⚠️ Channel id and cid are not the same thing 

```
cid = "\(type):\(id)"
```

CID is the only way to uniquely identify a channel (ie. "messaging:hello" and "team:hello" have same id, different type and cid)

Reply:
```suggestion
    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases), cid: ChannelId? = nil, _ callback: @escaping OnEvent) -> Subscription {
```